packages/x-date-pickers/src/CalendarPicker/CalendarPicker.test.tsx

```diff
@@ -13,7 +13,7 @@ import {
   wrapPickerMount,
   createPickerRenderer,
 } from 'test/utils/pickers-utils';
-import describeValidation from '@mui/x-date-pickers/internals/utils/describeValidation';
+import describeValidation from '@mui/x-date-pickers/tests/describeValidation';
```

Comment:
Should it be outside of the package ? 
In the `test/utils` folder ?

Reply:
Don't think so, because it's tests related to the package. testValidation will probably neither be used elsewhere

Reply:
The core had the pickers test in `test/utils` but that's of course something we can reconsider.
If we want to put `describeValidation` where you put it, mayber the `pickerUtils` should go there two i the future